src/routes/courses.ts

```diff
@@ -1,8 +1,9 @@
-import { DayOfWeek, Department, Term } from "@prisma/client";
```

Comment:
@forrest-keller do we want to import term or no?

Reply:
no